packages/app-bridge/src/tests/AppBridgeBlockStub.ts

```diff
@@ -189,6 +198,40 @@ export const getAppBridgeBlockStub = ({
             })
             .resolves({ assetBulkDownloadToken: 'token' }),
 
+        context: stub<Parameters<AppBridgeBlock['context']>>().callsFake((args) => {
+            if (args === undefined) {
+                return {
+                    get: () => [blockId, sectionId],
+                };
+            } else {
+                switch (args) {
+                    case 'blockId':
+                        return {
+                            get: () => blockId,
+                        };
+                    case 'sectionId':
+                        return {
+                            get: () => sectionId,
+                        };
+                    default:
+                        return {
+                            get: () => {
+                                throw new Error(`Unknown context key: ${args}`);
+                            },
+                        };
+                }
+            }
+        }),
+
+        subscribe: stub<Parameters<AppBridgeBlock['subscribe']>>().callsFake((eventName, callback) => {
+            if (eventName === 'assetsChosen') {
+                callback({ assets: [AssetDummy.with(123)] } as subscribeCallbackArguments);
```

Comment:
```suggestion
                callback({ assets: [AssetDummy.with(123)] } as Parameters<AppBridgeBlock['subscribe']>[1]);
```

Unfortunately this wasn't possible, as

`Argument of type '((eventReturn: { assets: Asset[]; }) => void) | ((eventReturn: { template: TemplateLegacy; }) => void) | ((eventReturn_0: Record<string, unknown>, eventReturn_1: Record<string, unknown>) => void) | ... 6 more ... | ((eventReturn_0: BlockContext, eventReturn_1: BlockContext) => void)' is not assignable to parameter of type 'Record<string, unknown> & { assets: Asset[]; } & { template: TemplateLegacy; } & BlockState & number & [number | undefined, number | undefined] & BlockContext'.
  Type '(eventReturn: { assets: Asset[]; }) => void' is not assignable to type 'Record<string, unknown> & { assets: Asset[]; } & { template: TemplateLegacy; } & BlockState & number & [number | undefined, number | undefined] & BlockContext'.
    Type '(eventReturn: { assets: Asset[]; }) => void' is not assignable to type 'Record<string, unknown>'.
      Index signature for type 'string' is missing in type '(eventReturn: { assets: Asset[]; }) => void'.ts(2345)
Conversion of type '{ assets: Asset[]; }' to type '((eventReturn: { assets: Asset[]; }) => void) | ((eventReturn: { template: TemplateLegacy; }) => void) | ((eventReturn_0: Record<string, unknown>, eventReturn_1: Record<string, unknown>) => void) | ... 6 more ... | ((eventReturn_0: BlockContext, eventReturn_1: BlockContext) => void)' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.ts(2352)`